src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,18 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/gatm
+server.port=8082
+spring.datasource.username=root
+spring.datasource.password=Glor!ous26
```

Comment:
can you pick secrets from environment variable instead. some like
spring.datasource.password=${DB_PASSWORD}